src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,101 @@
+import css from './Footer.module.css'
+
+export const Footer = (): JSX.Element => { 
+    return (
+        <footer className={css.footer}>
+            <div className={css.shadowBorder}></div>
+            <div className={css.box}>
+                <div className={css.column}>
+                    <h1 className={css.title}>Help</h1>
+                    <ul className={css.list}>
+                        <li className={css.item}>Payment</li>
```

Comment:
this is not a React way of writing code.

please create a mock constant, e.g. menu and map it here

const menu = [{.id: 0, href: '/payment', label: 'Payment' }]